cumulus/parachains/runtimes/assets/common/src/matching.rs

```diff
@@ -58,6 +58,24 @@ impl<SelfParaId: Get<ParaId>> ContainsPair<MultiLocation, MultiLocation>
 	}
 }
 
+pub struct FromNetwork<SelfNetworkId>(sp_std::marker::PhantomData<SelfNetworkId>);
```

Comment:
@claravanstaden I would rename `SelfNetworkId` -> `ExpectedNetworkId`, this is a different situation than with `SelfParaId` - `SelfParaId` is meant to be actual runtime's paraId, but here `SelfNetworkId` is meant to be remote networkId